sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md

```diff
@@ -28,6 +27,21 @@
 - Upgraded `azure-core` dependency to `1.11.0`.
 - Upgraded `azure-core-amqp` dependency to `2.0.0`.
 
+### Known issues 
+### Can not resolve 'BinaryData' 
+  Can not resolve 'BinaryData' When using 'azure-messaging-servicebus:7.0.0' and other Azure SDKs in the same pom.xml
+  file.
+- **Issue**: This compile time error occurs due to a known 'azure-core:1.11.0' dependency conflict between 
+  'azure-messaging-servicebus:7.0.0' and other Azure SDKs, such as 'azure-identity:1.2.0', which depend on older 
+  versions of azure-core. 
+  When two libraries depend on different versions of the same package, Maven will simply pick the first version that it 
+  sees and ignores the others.
+- **Resolution**: A workaround is to change the order of dependencies in the application's pom.xml file to list 
+  'azure-messaging-servicebus' before any other 'azure-*' client libraries. A Fix for the January 2021 release is being 
+  tracked at https://github.com/Azure/azure-sdk-for-java/issues/17942 .
+- **Additional Information**: More information on diagnosing and resolving dependency conflicts can be found at
+  https://github.com/Azure/azure-sdk-for-java/wiki/Frequently-Asked-Questions#dependency-conflict .
```

Comment:
I the link should be https://github.com/Azure/azure-sdk-for-java/wiki/Frequently-Asked-Questions#im-getting-a-nosuchmethoderror-or-noclassdeffounderror

Reply:
also there's a space between the link and the period on both line 41 and on line 43.

Reply:
fixed.